packages/slider/src/Slider.ts

```diff
@@ -548,4 +554,8 @@ export class Slider extends SizedMixin(ObserveSlotText(SliderHandle, ''), {
             }
         }
     }
+
+    protected override firstUpdated(changes: PropertyValues<this>): void {
+        super.firstUpdated(changes);
+    }
```

Comment:
If we're not actually doing work here than empty additions to the call stack are just costing us tiny amounts of performance.